src/FakeItEasy/ISequentialCallContext.cs

```diff
@@ -0,0 +1,23 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.ComponentModel;
+    using FakeItEasy.Core;
+
+    /// <summary>
+    /// Represents a context to make assertions on the order of the calls made on a fake.
+    /// </summary>
+    public interface ISequentialCallContext : IHideObjectMembers
+    {
+        /// <summary>
+        /// This method supports the FakeItEasy infrastructure and is not intended to be used directly from your code.

```

Comment:
I used the same wording as in the .NET Framework documentation, and they put it in the summary. I think it makes sense, because users rarely read as far as the remarks when they look at the doc (and they see only the summary in Intellisense tooltips, not the remarks). The first thing we want to communicate about this method is "it's not meant for you, don't use it". But if you think it's important, I can move it to `<remarks>`.